griptape/chat_local/chat_local.py

```diff
@@ -0,0 +1,41 @@
+from attr import define, field, Factory
+from typing import Any
+from griptape.structures import Agent
+from griptape.rules import Rule
+from griptape.drivers import LocalStructureRunDriver, LocalConversationMemoryDriver
+from griptape.tasks import StructureRunTask
+from griptape.memory.structure import ConversationMemory
+
+# Simple local agent example with conversation memory. 
+def build_agent():
+    return Agent(
+        conversation_memory=ConversationMemory(
+            driver=LocalConversationMemoryDriver(
+                file_path="conversation_memory.json"
+            )),
+        rules=[
+            Rule(
+                value = "You are an intelligent agent tasked with answering questions."
+            ),
+            Rule(
+                value = "All of your responses are less than 5 sentences."
+            )
+        ]
+        
+    )
+
+# Use this class in order to run local agents with Local Conversation Memory.
+@define
+class Chat_Local:
```

Comment:
did we make a decision not to use a consistent chat interface? If so, I'm OK, but otherwise I think an interface class with specialization makes sense here.